### PR TITLE
Coma voxel emission rendering for the opengl rendering path

### PR DESCRIPTION
**Pull request by oknuutti**
These changes make it possible to render comas when using the opengl rendering engine. At the moment voxel rendering happens in Python after OpenGL stuff. Thus, shadows are not taken into account. The coma must first be generated using the ComaCrator, then it can be loaded by adding e.g.
```
            "coma":
            {
                "file": "data/models/Jets--ROS_CAM1_20150710T074301.json",
                "intensity": 2e-6
            },
```
to the `sssb` section of an input file. The coma defining json file referenced is generated by the ComaCreator and is of form
```
{
  "dimensions": [40, 40, 40],
  "resolution": 600,
  "tiles_x": 25,
  "tiles_y": 25,
  "tiles_file": "data/models/Jets--ROS_CAM1_20150710T074301.exr"
}
```

I also moved the `brdf_params` reflectivity model parameters to be inside the `sssb` section instead of the `environment` section as it makes more sense that way.

### CODE REVIEW

File: data/input/oneshot_itokawa_opengl.json [hunk 1]
```diff
@@ -77,13 +64,25 @@
                 "file": "data/models/itokawa_f3145728.obj",
                 "name": "Itokawa"
             },
-            "albedo": 0.15,
```
Comment (YgabrielsY) [inline data/input/oneshot_itokawa_opengl.json:80]: Is it intended that the albedo value is deleted?

File: data/input/oneshot_67p_opengl.json [hunk 2]
```diff
@@ -76,13 +63,30 @@
                 "file": "data/models/67P_C-G_shape_model_MALMER_2015_11_20-in-km.obj",
                 "name": "67P"
             },
-            "albedo": 0.15,
```
Comment (YgabrielsY) [inline data/input/oneshot_67p_opengl.json:79]: Is it intended that the albedo value is deleted?

**Review by YgabrielsY — COMMENTED** [review]
Only one question in both changed definition files

File: data/input/oneshot_itokawa_opengl.json (hunk 1 above)
Comment (oknuutti) [reply data/input/oneshot_itokawa_opengl.json:80]: Dropping the albedo was intended as it's covered by the bidirectional reflectance distribution function parameters `brdf_params`. Hapke 2012 BRDF is the default model currently for OpenGL and there's a parameter there called single scattering albedo, which has a similar, although more fine-grained meaning.

File: data/input/oneshot_67p_opengl.json (hunk 2 above)
Comment (oknuutti) [reply data/input/oneshot_67p_opengl.json:79]: Referring to comment reply at `oneshot_itokawa_opengl.json`

File: data/input/oneshot_itokawa_opengl.json (hunk 1 above)
Comment (YgabrielsY) [reply data/input/oneshot_itokawa_opengl.json:80]: Alright, just wanted to be sure

**Review by YgabrielsY — APPROVED** [review]
Changes accepted

**Pull request merged**